force-app/main/default/triggers/AccountTrigger.trigger

```diff
@@ -0,0 +1,68 @@
+/* Account trigger should do the following:
+*  1. Set the account type to prospect.
+*  2. Copy the shipping address to the billing address.
+*  3. Set the account rating to hot.
+* 4. Create a contact for each account inserted.
+*/
+
+trigger AccountTrigger on Account (before insert, after insert) {
+    if (Trigger.isInsert) { // Only run on Insert
+        if (Trigger.isBefore) { // Check that the following actions will only run on Before Insert
+            for (Account a : Trigger.new) { // Loop through Accounts
+                /*
+                * Question 1
+                * When an account is inserted change the account type to 'Prospect' if there is no value in the type field.
+                */
+                if (String.isBlank(a.Type)) {
+                    a.Type = 'Prospect'; // Set Account Type if it is empty
+                }
+
+                /*
+                * Question 2
+                * When an account is inserted copy the shipping address to the billing address.
+                * BONUS: Check if the shipping fields are empty before copying.
```

Comment:
I dont think it is necessary to check that they are not blank, because if they are blank they will just copy over as blank. Probably better to make sure the shipping values are not blank (they were not supplied, therefore you can assume that the shipping is the same as the billing. Also you probably would not check one at a time since address is more a complete unit, you would not want to break it up.